README.md

```diff
@@ -106,10 +109,15 @@ The French locale is built from `source/locale/fr` with the following command:
 ```commandline
 make -e SPHINXOPTS="-D language='fr'" BUILDDIR="build/fr" html
 ```
+```
+in windows: sphinx-build source build/fr -D language='fr' -b html
+```
 
 - go to `build/fr/` 
 - run `index.html` in a browser of your choice
 
+
```

Comment:
Why so many blank lines?

Reply:
to my knowledge, I have not added any blank lines. Perhaps this might have been an issue when using vi? in any case, please see the new commit